examples/mackey_glass_parameter_jump.py

```diff
@@ -1,23 +1,20 @@
 """
-The following are a set of examples that implement the Mackey-Glass system 
-with a jump in the value of a parameter. Here we will change the value of β 
-from β0 to β1 at time t0. 
-
-The result of all the methods is basically identical and they range from the 
-most straightforward to more involved ones. Choose whichever one suits your 
-usecase. 
-    1. Integrating two separate models with diferent β values
-    2. Use β as a control parameter for a system amd modify its value midway 
-    through the integration
-    3. Use jitcdde.input
-    4. Use jitcxde_common.conditional
-    5. Use callbacks to define our own conditional
-    
-Note that if the parameter that needs to jump is the delay, you'll need to 
-manually set max_delay when defining the integrator in cases 2-5.
+The following are a set of examples that implement the Mackey-Glass system with a jump in the value of a parameter. Here we will change the value of β from β₀ to β₁ at time t₀. 
+
+The methods range from the most straightforward to more involved ones.
+
+    1. Integrate two separate models with diferent β values.
+    2. Use β as a control parameter for a system amd modify its value midway through the integration.
+    3. Use jitcdde.input.
+    4. Use jitcxde_common.conditional.
+    5. Use callbacks to define your own conditional.
+
+The result of all the methods is basically identical for this example, but not all may be feasible for your specific application – there is a reason that so many variants exist.
```

Comment:
Regarding this, if you have a decent idea regarding _when_ some of the examples might not be feasible, I think it'd be a nice idea to point those out.

Reply:
Indeed; done.